src/util/github-server.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_GITHUB_SERVER_URL = process.env.GITHUB_SERVER_URL || 'https://github.com';
```

Comment:
We would like any environment variable handling to happen at the entrypoint of running release-please and injecting the configuration into the classes rather than pulling from the environment in various spots.

In this case, can we configure/pull this from the `GitHub` class?

Reply:
maybe this should not happend in release-please but only in release-please-action while merging default...